src/javascript/EditPanelContainer/FormDefinitions/FormData.adapter.js

```diff
@@ -0,0 +1,74 @@
+import SelectorTypes from '../EditPanel/EditPanelContent/FormBuilder/SelectorTypes/SelectorTypes';
+
+const isDetailField = field => field.readOnly && field.targets.find(target => target.name === 'metadata');
+
+const getFields = (formDefinition, nodeData) => {
+    return formDefinition.fields
+        .filter(field => !isDetailField(field))
```

Comment:
Minor: you can use .filter(!isDetailField) directly as you did in the other function

Reply:
no you cannot, isDetailField is a function

Reply:
Mmmm, sorry I forget the "!"